js/Glacier2/callback/timeUtil.ts

```diff
@@ -0,0 +1,8 @@
+const millisecondsBeforeEpoch = 719162n * 24n * 60n * 60n * 1000n; // daysBeforeEpoch converted to milliseconds
+const ticksPerMillisecond = 10000n; // number of ticks per millisecond
+
+// Convert a timestamp representing the number of milliseconds elapsed since the epoch to a BigInt representing
```

Comment:
Should this be a doc-comment?

We should also say "Unix epoch". There is not one universal epoch. In the C++ code, we use system_clock's epoch, which is the Unix epoch (since C++20, but hopefully before as well).
See https://en.cppreference.com/w/cpp/chrono/system_clock